.gitignore

```diff
@@ -16,3 +16,7 @@ npm-debug.log*
 yarn.lock
 yarn-debug.log*
 yarn-error.log*
+
+babel.config.js
+jest.config.js
+
```

Comment:
Those files should always be included in the commits. Perhaps you used it locally (not really ideal), but now you are preventing others from committing those files.